### ISSUE TITLE
R6 Import bibliographic ID button not activating

### ISSUE THREAD

**Issue by mschnaars** [avalonmediasystem/avalon#1615]
Descriptive Summary:

iOS 9, Safari, on Spruce

When attempting to import bibliographic information after importing an item, after entering a bibliographic ID, the "Import" button should activate so that the correct info may be imported. The button does not activate until a save is attempted, after which the user will receive an error message. Only once this save is attempted will the button activate.

To reproduce:
Create an item, upload, save and continue, and enter bibliographic ID:
![first step](https://cloud.githubusercontent.com/assets/25825877/23032058/39e59ece-f441-11e6-97f7-d8a4dc876f45.jpg)

The import button does not yet work, so save and continue, hit enter, etc., and the page will return an error message due to required fields having no value, but the button is now activated:
![second step](https://cloud.githubusercontent.com/assets/25825877/23032094/5d839eb2-f441-11e6-9d64-dbbe8db473f2.jpg)

Hit import now and the fields will be populated:
![third step](https://cloud.githubusercontent.com/assets/25825877/23032144/882530ae-f441-11e6-8578-c9841f134dc0.jpg)

**Comment by phuongdh** [comment]
Is problem also present on Safari OSX?

**Comment by jlhardes** [comment]
Verified the import button works on Firefox 51 and Safari 10 on OS 10.12.3.

**Comment by cjcolvar** [comment]
@avalonmediasystem/avalon-product-owners I couldn't reproduce this on spruce.  I recommend closing it.

**Comment by joncameron** [comment]
Closing as per Chris' comment. We'll write a new issue if this reappears.
